qemu/tests/cfg/blockdev_commit_to_nospace.cfg

```diff
@@ -0,0 +1,49 @@
+- blockdev_commit_to_nospace:
+    type = blockdev_commit_to_nospace
+    virt_test_type = qemu
+    only Linux
+    device_tag = 'data'
+    images += " ${device_tag}"
+    force_create_image_data = yes
+    remove_image_data = yes
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 2G
+    snapshot_tags_data = sn1
+
+    image_size_sn1 = 2G
+    image_format_sn1 = qcow2
+    image_name_sn1 = sn1
+
+    rebase_mode = unsafe
+    qemu_force_use_drive_expression = no
+
+    tmpdir = /home/tmp
+    mount_dir = /mnt/test
+    loopdev = ${tmpdir}/loopdev
+    backend_base = ${tmpdir}/disk.img
+    image_name_data = ${mount_dir}/data
+    vgname = test
+    lvname = testlvm1
+    pre_command = "mkdir -p ${tmpdir} && losetup -f > ${loopdev} &&"
+    pre_command += "qemu-img create -f raw ${backend_base} 4G &&"
+    pre_command += "losetup `cat ${loopdev}` ${backend_base} &&"
+    pre_command += "pvcreate `cat ${loopdev}` &&"
+    pre_command += "vgcreate ${vgname} `cat ${loopdev}` &&"
+    pre_command += "lvcreate -L 100M -n ${lvname} ${vgname} &&"
+    pre_command += "mkfs -t ext3 /dev/${vgname}/${lvname} && mkdir -p ${mount_dir} &&"
+    pre_command += "mount /dev/${vgname}/${lvname} ${mount_dir}"
+
+    post_command += "umount ${mount_dir} &&"
+    post_command += "lvremove -f /dev/${vgname}/${lvname} &&"
+    post_command += "vgremove -f ${vgname} &&"
+    post_command += "pvremove -f `cat ${loopdev}` &&"
+    post_command += "losetup -d `cat ${loopdev}` &&"
+    post_command += "rm -rf ${mount_dir} &&"
```

Comment:
Is it possible to simplify the commands? e.g. 

source_path = /tmp/tmp_target_path
     images_base_dir_data = ${source_path}   >>> data will be created under source_path, so we can set image_name_data = data instead of a abs path
     tmp_image_file = /tmp/tmp_image_file
     pre_command = "mkdir -p ${source_path} && dd if=/dev/urandom of=${tmp_image_file} bs=1M count=100 &&"
     pre_command += " mkfs.ext4 ${tmp_image_file} && mount -o loop ${tmp_image_file} ${source_path}"
     post_command = "umount -f ${source_path} && rm -rf ${tmp_image_file} ${source_path}"

Don't use lvm, which might be simple
The comment is also applied for #2973 #2972 , all of these PRs don't need extending space, so we may not need lvm.

Reply:
Have updated, thanks @zhencliu